From 4c7020ec299685e92b124ccc32cae37f5f64684f Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 31 Mar 2021 08:16:04 +0200
Subject: [PATCH 1/1] efi_loader: Uart device path missing break

In the devicepath node to text conversion for Uart nodes a break statement
is missing.

Indicated by Coverity Scan CID 330038
Fixes: 62df6e9c9994 ("efi_loader: Uart device path")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_device_path_to_text.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/efi_loader/efi_device_path_to_text.c b/lib/efi_loader/efi_device_path_to_text.c
index 43554cd771..675e80bcb8 100644
--- a/lib/efi_loader/efi_device_path_to_text.c
+++ b/lib/efi_loader/efi_device_path_to_text.c
@@ -126,8 +126,10 @@ static char *dp_msging(char *s, struct efi_device_path *dp)
 		switch (uart->stop_bits) {
 		case 2:
 			s += sprintf(s, "1.5)");
+			break;
 		default:
 			s += sprintf(s, "%d)", uart->stop_bits);
+			break;
 		}
 		break;
 	}
-- 
2.30.2

